pkg/path/finch_linux.go

```diff
@@ -34,6 +34,11 @@ func (fp Finch) BuildkitSocketPath() string {
 	return filepath.Join(fp.FinchRuntimeDataDir(), "buildkit", "buildkitd.sock")
 }
 
+// DockerConfigPath returns the path to the docker config file.
+func (fp Finch) DockerConfigPath() string {
```

Comment:
DockerConfigDir?

Also, should we add a comment here that nerdctl uses `${DOCKER_CONFIG}/config.json` for the authentication with image registries.

Reply:
Fixed in latest revision